app/javascript/components/VenueEdit.jsx

```diff
@@ -0,0 +1,98 @@
+import React, { Component } from 'react';
+import axios from 'axios';
+
+export default class VenueEdit extends Component {
+  constructor(props) {
+    super(props);
+    
+    this.state = {
+      address: "",
+      decription: "",
+      people_capacity: "",
+      creationErrors: ""
+    };
+
+    this.handleChange = this.handleChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  handleChange(event) {
+    this.setState({
+      [event.target.name]: event.target.value
+    });
+  }
+
+  handleSubmit(event) {
+    const { address, description, people_capacity } = this.state;
```

Comment:
Why you need this const? If only for data in axios.put, you can use this.state instead of
```ruby
address: address,
description: description,
people_capacity: people_capacity
```

Reply:
yeap, done